Soccar [Library]/src/nl/soccar/library/enumeration/MapType.java

```diff
@@ -7,8 +7,31 @@
  */
 public enum MapType {
     
-    GRASSLAND,
-    MOON,
-    DESERT
+    GRASSLAND(0),
+    MOON(1),
+    DESERT(2);
+    
+    private final int id;
+
+    private MapType(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static MapType parse(int id) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)